src/main/java/hudson/plugin/versioncolumn/JVMVersionComparator.java

```diff
@@ -91,9 +91,16 @@ private int getMasterBytecodeMajorVersionNumber() {
 
     private boolean isAgentRuntimeCompatibleWithJenkinsBytecodeLevel(String agentMajorMinorVersion) {
         Integer masterBytecodeLevel = getMasterBytecodeMajorVersionNumber();
-        Integer agentVMMaxBytecodeLevel = JDK_VERSION_NUMBER_TO_BYTECODE_LEVEL_MAPPING.get(agentMajorMinorVersion);
-
-        return masterBytecodeLevel <= agentVMMaxBytecodeLevel;
+        try {
+            Integer agentVMMaxBytecodeLevel = JDK_VERSION_NUMBER_TO_BYTECODE_LEVEL_MAPPING.get(agentMajorMinorVersion);
+            return masterBytecodeLevel <= agentVMMaxBytecodeLevel;
+        } catch(NullPointerException e) {
```

Comment:
Use Null check instead

Reply:
it's not returning `null` but throwing an NPE. I'll go for @Wadeck idea and use `contains`.

Reply:
As always, you're right. I change the code for that.